### DISCUSSION TITLE
Getting this error while installing the latest eclipse plugin

### DISCUSSION THREAD

**Discussion by gab444** [dbeaver/dbeaver#14164] [Q&A] [answered]
@serge-rider :getting this error while installing the latest eclipse plugin, is there a way to install previous eclipse versions?
![image](https://user-images.githubusercontent.com/69998999/135355109-21ede41a-44d4-47a1-b086-a5f2f8d221cc.png)

_Originally posted by @gab444 in https://github.com/dbeaver/dbeaver/issues/7358#issuecomment-930578678_

**Comment by Matvey16** [comment]
Hello @gab444 I see that you are installing 21.2.0 and it is not the latest version. Can you clarify how you are installing it and also what version of Eclipse do you use?

**Comment by gab444** [comment]
@Matvey16 
I am using following url to install the plugin in eclipse 
https://dbeaver.com/update/ee/latest/ 

![image](https://user-images.githubusercontent.com/69998999/136272532-fcf8e24b-b26b-4f78-9dda-c0647c5936e2.png)

I am on eclipse version 4.6.3

**Comment by Matvey16** [comment]
@gab444  Unfortunately, due to recent updates Dbeaver may no longer be compatible with older versions of Eclipse. Have you tried installing it for a more recent versions? If you need to use older Eclipse version, the only solution is to use older Dbeaver version.

**Comment by gab444** [comment]
@Matvey16 
Thanks for your response. 
I am ok with older eclipse version, as I cannot install newer version of the eclipse. is there a way to install older eclipse plugins ?

**Comment by Matvey16** [comment, marked as answer]
@gab444 You can use this URL https://dbeaver.com/files/7.3.0/repository-ee/ in `Work with` field in Eclipse where `7.3.0` is version number and you can change the url with other version. Availible versions can be found here https://dbeaver.com/files/

**Comment by gab444** [comment]
@Matvey16 thank you. it worked. I was able to install dbeaver in eclipse.
